vignettes/getting_started.Rmd

```diff
@@ -72,27 +72,28 @@ manual_recodes
 This list can be edited manually to add the recoded values in `newValues`. For example, you can save the file as an `.xlsx` file and edit it in excel, then load it into R again:
 
 ```{r, eval = FALSE}
-openxlsx::write_xlsx(manual_recodes, "./manual_recodes.xlsx")
-recoded_df2 <- openxlsx::read_xlsx("./manual_recodes_changed.xlsx")
+openxlsx::write.xlsx(manual_recodes, "./manual_recodes.xlsx")
+recoded_df2 <- openxlsx::read.xlsx("./manual_recodes_changed.xlsx")
 ```
 
-Or, you can edit the data directly in R:
+You can also edit the data directly in R:
 ```{r}
 manual_recodes$newValues <- c("Egypt", "UK")
 ```
 
-## 3. Update your recode list 
+## 3. Update Your Recode List 
 The next step is to update your recode data base, so it contains all needed values:
 ```{r}
 updateRecodeDB(newRecodes = manual_recodes,
                             oldValues = "country",
                             recodeDBPath = here::here("tests", "testthat", "helper_recodeDB.xlsx"), # I use the here package to set the path to the data base
                             newRecodeDBPath = here::here("tests", "testthat", "helper_recodeDB_2.xlsx"), 
                             name = "country")
+# is it supposed to return NULL?
```

Comment:
Yes! This function updates a data base, so it doesn't need to return anything in R. 
(Please remove this comment).

Reply:
done. thanks for reminding